site/_config.yml

```diff
@@ -32,3 +32,7 @@ gems:
   - jekyll-seo-tag
   - jekyll-avatar
   - jekyll-mentions
+
+exclude:
+  - README.md
+  - .gitignore
```

Comment:
@parkr sounds that `.gitignore` should be exclude by `EntryFilter`. I will inspect this later.